test/fixtures/keys/agent6.cnf

```diff
@@ -0,0 +1,18 @@
+[ req ]
+string_mask            = utf8only
+utf8                   = yes
+default_bits           = 1024
+days                   = 999
+distinguished_name     = req_distinguished_name
+attributes             = req_attributes
+prompt                 = no
+
+[ req_distinguished_name ]
+C                      = HU
+L                      = Budapest
+O                      = Tresorit
+CN                     = Ádám Lippai
```

Comment:
@shigeki I don't see how it matters, so I'd rather leave it as it is now. We have lots of junk in other certs too atm.